src/ipa/libipa/exposure_mode_helper.cpp

```diff
@@ -166,8 +166,15 @@ ExposureModeHelper::splitExposure(utils::Duration exposure) const
 		return { minShutter_, minGain_, exposure / (minShutter_ * minGain_) };
 
 	utils::Duration shutter;
-	double stageGain;
 	double gain;
+	/*
+	 * From here on all we can do is max out the shutter time, followed by
```

Comment:
Does this comment still make sense when it's moved up? It reads like it has a positional context "From here on..." which sounds to me like it implies it might only make sense /after/ the loop which now follows with this patch...